draft-wussler-openpgp-pqc.md

```diff
@@ -319,8 +319,10 @@ In contrast, the hashed-based signature scheme SPHINCS+ is considered to be
 sufficiently well understood with respect to its security assumptions in order
 to be used standalone. To this end, this document specifies the following new
 set: SPHINCS+ standalone and CRYSTALS-* as composite with ECC-based KEM and
-digital signature schemes. Here, the term "composite" indicates that the
-combination of the two components forms a single atomic object.
+digital signature schemes. Here, the term "composite" indicates that any data
+structure or algorithm pertaining to the 
+combination of the two components appears as single data structure or algorithm
+from protocol perspective.
```

Comment:
Is there a 'the' missing?, i.e. should it read 'from the protocol perspective'?

Reply:
Indeed, added the missing 'the'.